client/state/selectors/get-account-recovery-reset-user-first-name.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * @param {Object} state Global app state
+ * @return {String} The first name of the user making password reset request.
+ */
+export default ( state ) => {
+	return get( state, 'accountRecovery.reset.userData.firstName', '' );
```

Comment:
Here is how the UI looks:
![image](https://cloud.githubusercontent.com/assets/1842898/23446139/ae2b4eca-fe7b-11e6-8375-062ae4be1fce.png)
Those text fields are `<input>` and passing `null` triggers some warnings for them. I don't want to do `null` check in the component level so I default them as empty strings.